spec/feature/devise/sign_in_spec.rb

```diff
@@ -1,14 +1,58 @@
 require 'rails_helper'
 
 feature 'User signed in' do
-  given(:user) { build(:user) }
-  scenario 'User try to sign in' do
-    visit root_path
-    mock_auth user
-    click_link_or_button 'sign in (GitHub)'
-
-    expect(page).to have_content 'Signed in successfully.'
-    expect(current_path).to eq root_path
+  context 'new user' do
+    given(:user) { build(:user) }
+
+    context 'from whitelisted organisation' do
+      scenario 'try to sign in' do
+        visit root_path
+        mock_auth user
+        click_link_or_button 'sign in (GitHub)'
+
+        expect(page).to have_content 'Signed in successfully.'
+        expect(current_path).to eq root_path
+      end
+    end
+
+    context 'not from whitelisted organisation' do
```

Comment:
Once again, what's the difference between these two contexts?
I don't like test assumptions being implicit here, please find a way to make this organisational membership a `given` statement in your tests